paimon-core/src/main/java/org/apache/paimon/disk/ExternalBuffer.java

```diff
@@ -44,6 +44,9 @@ public class ExternalBuffer implements RowBuffer {
 
     private static final Logger LOG = LoggerFactory.getLogger(ExternalBuffer.class);
 
+    private static final Lz4BlockCompressionFactory compactionFactory =
```

Comment:
You can use `BlockCompressionFactory` to pass `compression` to here.

Reply:
Resolved